packages/contracts/contracts/L2/ValidatorRewardVault.sol

```diff
@@ -144,6 +144,29 @@ contract ValidatorRewardVault is FeeVault, ISemver {
         require(success, "ValidatorRewardVault: ETH transfer failed");
     }
 
+    /**
+     * @notice Withdraws all remaining contract balance excluding reserved amount to ProtocolVault.
+     *         Reverts if the withdraw amount is less than the minimum withdrawal amount.
+     */
+    function withdrawToProtocolVault() external {
+        uint256 amount = address(this).balance - totalReserved;
+        require(
+            amount >= MIN_WITHDRAWAL_AMOUNT,
+            "ValidatorRewardVault: withdrawal amount must be greater than minimum withdrawal amount"
+        );
+        require(
+            amount > 0,
+            "ValidatorRewardVault: withdrawal amount must be greater than zero"
+        );
+
+        totalProcessed += amount;
```

Comment:
How about applying unchecked to the line to reduce gas costs, similar to the way it's done in other functions?